### ISSUE TITLE
Implementing Tracker

### ISSUE THREAD

**Issue by RohaanA** [yohanshin/WHAM#76]
Hi @yohanshin ,

I've been trying out different videos using WHAM, and so far the results are always good. However sometimes I have noticed that WHAM can miss a person for a frame or two. Obviously this is not an issue for me, but when I am converting the poses into blender it separates the poses into different IDs. 

https://github.com/yohanshin/WHAM/assets/75722072/9b5c2c8d-c630-4c9a-83c7-5830511b7b7e

Example in this video, if you look at the far side player the model fluctuates sometimes. So when I am converting the poses into Fbx it changes their IDs. 

The problem is that since the IDs change I cannot automate the fbx generation part. Do you think a tracker could be implemented here? Especially since yolov8 natively supports trackers now.

**Comment by yohanshin** [comment]
Yes, if better person tracker can be implemented, it would be great. I think Human4D provides good tracking performance.

**Comment by g8392** [comment]
> Hi @yohanshin ,
> 
> I've been trying out different videos using WHAM, and so far the results are always good. However sometimes I have noticed that WHAM can miss a person for a frame or two. Obviously this is not an issue for me, but when I am converting the poses into blender it separates the poses into different IDs.
> download.-.Copy.mp4
> 
> Example in this video, if you look at the far side player the model fluctuates sometimes. So when I am converting the poses into Fbx it changes their IDs.
> 
> The problem is that since the IDs change I cannot automate the fbx generation part. Do you think a tracker could be implemented here? Especially since yolov8 natively supports trackers now.

I'm also trying to export into a DCC, could you please share your method? Thanks

**Comment by yang1hu** [comment]
> Hi @yohanshin ,
> 
> I've been trying out different videos using WHAM, and so far the results are always good. However sometimes I have noticed that WHAM can miss a person for a frame or two. Obviously this is not an issue for me, but when I am converting the poses into blender it separates the poses into different IDs.
> 
>  download.-.Copy.mp4 
> Example in this video, if you look at the far side player the model fluctuates sometimes. So when I am converting the poses into Fbx it changes their IDs.
> 
> The problem is that since the IDs change I cannot automate the fbx generation part. Do you think a tracker could be implemented here? Especially since yolov8 natively supports trackers now.

hi,would you please tell me the method you import the mesh  into blender. I wanna display it in blender.thank you

**Comment by RohaanA** [comment]
> > Hi @yohanshin ,
> > I've been trying out different videos using WHAM, and so far the results are always good. However sometimes I have noticed that WHAM can miss a person for a frame or two. Obviously this is not an issue for me, but when I am converting the poses into blender it separates the poses into different IDs.
> > download.-.Copy.mp4
> > Example in this video, if you look at the far side player the model fluctuates sometimes. So when I am converting the poses into Fbx it changes their IDs.
> > The problem is that since the IDs change I cannot automate the fbx generation part. Do you think a tracker could be implemented here? Especially since yolov8 natively supports trackers now.
> 
> hi,would you please tell me the method you import the mesh into blender. I wanna display it in blender.thank you

Yup, so basically I am using a script that was designed for VIBE.  (You can go to VIBE's github page and find it). 

Since WHAM was improved upon TCMR, and TCMR was built upon VIBE, the same script also works on WHAM.

**Comment by yang1hu** [comment]
> > > Hi @yohanshin ,
> > > I've been trying out different videos using WHAM, and so far the results are always good. However sometimes I have noticed that WHAM can miss a person for a frame or two. Obviously this is not an issue for me, but when I am converting the poses into blender it separates the poses into different IDs.
> > > download.-.Copy.mp4
> > > Example in this video, if you look at the far side player the model fluctuates sometimes. So when I am converting the poses into Fbx it changes their IDs.
> > > The problem is that since the IDs change I cannot automate the fbx generation part. Do you think a tracker could be implemented here? Especially since yolov8 natively supports trackers now.
> > 
> > 
> > hi,would you please tell me the method you import the mesh into blender. I wanna display it in blender.thank you
> 
> Yup, so basically I am using a script that was designed for VIBE. (You can go to VIBE's github page and find it).
> 
> Since WHAM was improved upon TCMR, and TCMR was built upon VIBE, the same script also works on WHAM.

ok! thank you very much!

**Comment by yang1hu** [comment]
> > > Hi @yohanshin ,
> > > I've been trying out different videos using WHAM, and so far the results are always good. However sometimes I have noticed that WHAM can miss a person for a frame or two. Obviously this is not an issue for me, but when I am converting the poses into blender it separates the poses into different IDs.
> > > download.-.Copy.mp4
> > > Example in this video, if you look at the far side player the model fluctuates sometimes. So when I am converting the poses into Fbx it changes their IDs.
> > > The problem is that since the IDs change I cannot automate the fbx generation part. Do you think a tracker could be implemented here? Especially since yolov8 natively supports trackers now.
> > 
> > 
> > hi,would you please tell me the method you import the mesh into blender. I wanna display it in blender.thank you
> 
> Yup, so basically I am using a script that was designed for VIBE. (You can go to VIBE's github page and find it).
> 
> Since WHAM was improved upon TCMR, and TCMR was built upon VIBE, the same script also works on WHAM.

Thank you for your hits! I have display the mesh in blender. however I found that the person has been locked in (0,0,0); do you have the same problem

**Comment by g8392** [comment]
> > > Hi @yohanshin ,
> > > I've been trying out different videos using WHAM, and so far the results are always good. However sometimes I have noticed that WHAM can miss a person for a frame or two. Obviously this is not an issue for me, but when I am converting the poses into blender it separates the poses into different IDs.
> > > download.-.Copy.mp4
> > > Example in this video, if you look at the far side player the model fluctuates sometimes. So when I am converting the poses into Fbx it changes their IDs.
> > > The problem is that since the IDs change I cannot automate the fbx generation part. Do you think a tracker could be implemented here? Especially since yolov8 natively supports trackers now.
> > 
> > 
> > hi,would you please tell me the method you import the mesh into blender. I wanna display it in blender.thank you
> 
> Yup, so basically I am using a script that was designed for VIBE. (You can go to VIBE's github page and find it).
> 
> Since WHAM was improved upon TCMR, and TCMR was built upon VIBE, the same script also works on WHAM.

Could you guys

> > > > Hi @yohanshin ,
> > > > I've been trying out different videos using WHAM, and so far the results are always good. However sometimes I have noticed that WHAM can miss a person for a frame or two. Obviously this is not an issue for me, but when I am converting the poses into blender it separates the poses into different IDs.
> > > > download.-.Copy.mp4
> > > > Example in this video, if you look at the far side player the model fluctuates sometimes. So when I am converting the poses into Fbx it changes their IDs.
> > > > The problem is that since the IDs change I cannot automate the fbx generation part. Do you think a tracker could be implemented here? Especially since yolov8 natively supports trackers now.
> > > 
> > > 
> > > hi,would you please tell me the method you import the mesh into blender. I wanna display it in blender.thank you
> > 
> > 
> > Yup, so basically I am using a script that was designed for VIBE. (You can go to VIBE's github page and find it).
> > Since WHAM was improved upon TCMR, and TCMR was built upon VIBE, the same script also works on WHAM.
> 
> Thank you for your hits! I have display the mesh in blender. however I found that the person has been locked in (0,0,0); do you have the same problem

Hello have you figured it out? Could you share the script?
